src/sentry/static/sentry/app/views/eventsV2/data.tsx

```diff
@@ -11,12 +11,12 @@ import UserBadge from 'app/components/idBadge/userBadge';
 import getDynamicText from 'app/utils/getDynamicText';
 import overflowEllipsis from 'app/styles/overflowEllipsis';
 import pinIcon from 'app/../images/graph/icon-location-filled.svg';
-import {Organization, NewQuery} from 'app/types';
+import { Organization, NewQuery } from 'app/types';
```

Comment:
You might want to check that your editor is using the same prettier version/configuration as our application does. These whitespace changes will likely enrage prettier.

Reply:
Yeah noted. I'll upgrade.